### ISSUE TITLE
Hover text for "test scores" in the selectivity section is wrong

### ISSUE THREAD

**Issue by alex** [18F/college-choice#993]
It appears to have the hover text for the Graduation Rate section instead. See this screenshot:

![screen shot 2015-09-02 at 6 28 14 pm](https://cloud.githubusercontent.com/assets/772/9646315/663c9364-51a0-11e5-904b-ff02ca405420.png)

**Comment by LisaGee** [comment]
@jjoteal Can you confirm that this one has been resolved?

**Comment by jjoteal** [comment]
Yes, the tooltip text for Test Scores has been updated in glossary.yaml and is:

"At schools that report their test scores, 50 percent had scores within this range. Of the remaining 50 percent, half were above the range and half were below. Not all schools require test scores."

I also checked the school index file and it is referencing the correct tooltip (test-scores). 

So once things merge, we should be ok!
